R/tokens_ngrams.R

```diff
@@ -81,17 +81,7 @@ tokens_ngrams <- function(x, n = 2L, skip = 0L, concatenator = "_") {
 #' @noRd
 #' @export
 tokens_ngrams.character <- function(x, n = 2L, skip = 0L, concatenator = "_") {
-    # trap condition where a "text" is a single NA
```

Comment:
I'm replacing the original behaviour since this is not intended to run tokens on a character - it only works on vectors of characters (tokens).